doc/source/whatsnew/v1.2.0.rst

```diff
@@ -313,6 +313,7 @@ Performance improvements
   avoiding creating these again, if created on either. This can speed up operations that depend on creating copies of existing indexes (:issue:`36840`)
 - Performance improvement in :meth:`RollingGroupby.count` (:issue:`35625`)
 - Small performance decrease to :meth:`Rolling.min` and :meth:`Rolling.max` for fixed windows (:issue:`36567`)
+- Reduced peak memory usage in DataFrame.to_pickle() when using protocol=5 in python 3.8+
```

Comment:
can you make this ``` :meth:`DataFrame.to_pickle` ``` and put protocol=5 in double back-ticks so they will render; also add the issue number